jquery.floatThead.js

```diff
@@ -705,6 +706,13 @@
             }
             left = tableOffset.left - windowLeft;
           }
+          if (top < 0) {
+            // in some fixed layout's the top offset can return a negative, in which case using
+            // the last known good value works 100%
+            top = goodTop;
```

Comment:
wouldnt it be better to set it to 0 here? and then you can remove most of this logic